app/model/application.js

```diff
@@ -185,6 +197,26 @@ ApplicationSchema.statics.syncIssuesToGitHub = function(application) {
     .then(() => application);
 }
 
+ApplicationSchema.statics.syncLabelToGitHub = function(application) {
+  const fullName = application.github.fullName;
+  const label = application.label;
+  return gh.request(`GET /repos/${fullName}/labels/${label}`, {
+    token: application.github.APItoken,
+  }).catch(() => {
+    return gh.request(`POST /repos/${fullName}/labels`, {
+      token: application.github.APItoken,
+      body: {
+        name: application.label,
+        color: '3A416F',
+      },
+    })
+  }).then(labelResult => {
+    application.label = labelResult.name;
+    return application
+  })
+  .catch(() => application);
```

Comment:
The function? To sync the Github issue label :)
The catch? Not sure why I put that there.